app/models/manageiq/providers/ibm_cloud/vpc/network_manager/cloud_network.rb

```diff
@@ -1,2 +1,24 @@
 class ManageIQ::Providers::IbmCloud::VPC::NetworkManager::CloudNetwork < ::CloudNetwork
+  include ProviderObjectMixin
+
+  supports :delete do
+    if ext_management_system.nil?
+      unsupported_reason_add(:delete_cloud_network, _("The Cloud Network is not connected to an active %{table}") % {
+        :table => ui_lookup(:table => "ext_management_systems")
+      })
+    end
+  end
+
+  def raw_delete_cloud_network(_options)
```

Comment:
If you aren't going to use the options best to make it optional like, `_options = {}` so the caller doesn't have to pass anything